<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTagDialog</class>
 <widget class="QDialog" name="NewTagDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Tag</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="topFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="comboBoxVLayout">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <layout class="QGridLayout" name="comboBoxGridLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="1">
           <widget class="QComboBox" name="protocolComboBox">
            <property name="maxVisibleItems">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="protocolLabel">
            <property name="text">
             <string>Protocol Name</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="tagInputHLayout">
          <item>
           <widget class="QLabel" name="tagLabel">
            <property name="text">
             <string>Select a tag</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="tagComboBox">
            <property name="maxVisibleItems">
             <number>20</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="topFrameLine">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="generalGroupBox">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="title">
         <string>General Properties</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="tagNameHLayout">
           <item>
            <widget class="QLabel" name="tagNameLabel">
             <property name="text">
              <string>Tag Name             </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tagNameLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="descriptionHLayout">
           <item>
            <widget class="QLabel" name="descriptionLabel">
             <property name="text">
              <string>Description           </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="descriptionLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="destTypeLabel">
             <property name="text">
              <string>Dest. Datatype     </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="destTypeLineEdit">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="tagTypeHLayout">
           <item>
            <widget class="QLabel" name="tagTypeLabel">
             <property name="text">
              <string>Tag Type               </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tagTypeLineEdit">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="lengthHLayout">
           <item>
            <widget class="QLabel" name="lengthLabel">
             <property name="text">
              <string>Length                 </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lengthLineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="justifyLabel">
             <property name="text">
              <string>Justify With     </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="justifyLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="editableHLayout">
           <item>
            <widget class="QLabel" name="editableLabel">
             <property name="text">
              <string>Editable                </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="editableYesRButton">
             <property name="text">
              <string>Yes</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="editableNoRButton">
             <property name="text">
              <string>No</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>38</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="messageBasedGroupBox">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Message Based Properties</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="reqHLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="requiredLabel">
             <property name="text">
              <string>Required               </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="reqYesRButton">
             <property name="text">
              <string>Yes</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="reqNoRButton">
             <property name="text">
              <string>No</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="reqHSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>38</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="offsetLabel">
             <property name="text">
              <string>   Tag Offset </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="offsetLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="seqNumberHLayout">
           <item>
            <widget class="QLabel" name="sequenceNoLabel">
             <property name="text">
              <string>Sequence Number</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="seqNumberLineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="defaultValLabel">
             <property name="text">
              <string>Default Value</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="defaultValLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="possibleValHLayout">
           <item>
            <widget class="QLabel" name="possibleValLabel">
             <property name="text">
              <string>Possible Values      </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="possibleValComboBox"/>
           </item>
           <item>
            <spacer name="possibleValHSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>298</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
     <zorder>topFrameLine</zorder>
     <zorder>messageBasedGroupBox</zorder>
     <zorder>generalGroupBox</zorder>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="buttonFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="buttonFrameVLayout">
      <item>
       <widget class="Line" name="buttonFrameLine">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonFrameChildHLayout">
        <item>
         <spacer name="buttonsHorizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>338</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="createButton">
          <property name="text">
           <string>Create</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
